ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/GroupParametersLookup.java

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.utxo;
```

Comment:
_Leaving this as a note to myself, so don't worry about it for now._

Should group parameters be available for the consensual ledger as well? If the answer is yes, then `GroupParametersLookup` should either live in the `membership` module or the `ledger-common` module.

There is an argument it should be in the `membership` module regardless.

Reply:
@blsemo @dimosr @charlieR3 let me know if you have any thoughts.

Reply:
I think adding anything to Consensual Ledger is a product question.
And as I heard, Consensual ledger is not one of the top priorities.
On the other hand, this module does not have anything UTXO-specific.
Also, future ledger modules may need it.

Reply:
Yeah, that's my thinking as well.

Reply:
Potentially this should actually go into `application` along side the `MemberLookup` interface.

Reply:
Yes, I think it could fit in `application` with the `MemberLookup` interface. The `GroupParameters` interface is in the `membership` module already. 

Saying that though, the `NotaryInfo` interface is in `membership` but `NotaryLookup` is in `ledger-common`. I think `NotaryInfo` is mainly in `membership` though because `GroupParameters` depends on it and we don't want `membership` to depend on `ledger`.

I would lean towards having it in `membership` so that it is not ledger specific. We allow adding any custom metadata into to group parameters for example which doesn't necessarily need to be ledger specific.

Reply:
Maybe we should consider moving `NotaryLookup` too (we'd have to deprecate the existing one and copy it to `application`/`membership`).

I think feels like the right fit for `GroupParametersLookup` and possibly `NotaryLookup`.

Happy to go with the majority on this.

Reply:
The original logic for `NotaryLookup` being in `ledger-common` was that it's tightly linked with the ledger layer so it seems like a better place for it. But not sure that matters much if `NotaryInfo` is in `membership` anyway. `membership` is a bit of a cross layer concept anyway so I'm happy to go with the majority also.

Reply:
I would handle NotaryLookup in a separate ticket.
But I would move `GroupParametersLookup` to the most future-proof place in this one.
(However, I am not too sure whether application or membership is the best.)

Reply:
Lets move `GroupParametersLookup` to `v5.application.membership`. We can raise a tech debt for `NotaryLookup`, ideally to be reviewed before 5.1. Then we can decide if they should all move to the `membership` module or if they all live in `v5.application.membership`.